_posts/blog/2017-03-09-public_speaking_intros.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "About Intros, Nerves and Discovering Yourself"
+description: Public speaking allowed me to get to know myself, what works for me and what not. Here's what I learned about intros.
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Public speaking allowed me to get to know myself, what works for me and what not. Here's what I learned about intros.
+tags: [Public speaking, Speaking]
+image:
+date: 2017-03-08T00:39:55-04:00
+---
+
+Public speaking pushed me out of a lot of my comfort zones. I got to learn not only about the content that I was presenting and about the delivery, but also about myself. I found out how I react in certain situations, what feels genuine and what doesn’t, and what I should work on more.
+<br/>
+
+> *Public speaking is a journey of discovering yourself.*
+
+<br/>
+For example, I learned that I can control my speech rate and can use that to improve the effect of my words. But most of all, I learned that indeed, public speaking is one of humanity’s biggest fears, but that I can restrain it and decrease it and that hopefully, one day, I can transform it into contagious energy and enthusiasm for my audience. 
+
+I saw that before my talk, doing breathing exercises and a power pose stabilises my heart rate and reduces my nervousness. But, once I start talking, I still need a few minutes until I’m calm, and get in my speaking flow. My voice still shakes a bit.
```

Comment:
""Before, my voice still shakes a bit."? - it's not "before" but rather in those initial few minutes, my voice still shakes.